src/App.vue

```diff
@@ -0,0 +1,253 @@
+<script setup>
+import { ref, onMounted } from 'vue'
+
+const origins = ref([])
+const inputText = ref('')
+const placeholderURL = ref('https://hoppscotch.io')
+const errorMessage = ref('')
+
+const DEFAULT_ORIGIN_LIST = ["https://example.com", "https://another-example.com"]
```

Comment:
our `DEFAULT_ORIGIN_LIST` is just `https://hoppscotch.io`. its defined in the `src/defaultOrigins.ts`

Reply:
Resolved, imported `DEFAULT_ORIGIN_LIST` from `src/defaultOrigins.ts` will raise a commit on this PR soon.